FullyNoded/BitSense.xcdatamodeld/BitSense.xcdatamodel/contents

```diff
@@ -9,6 +9,7 @@
         <attribute name="cert" optional="YES" attributeType="Binary"/>
         <attribute name="id" optional="YES" attributeType="UUID" usesScalarValueType="NO"/>
         <attribute name="isActive" optional="YES" attributeType="Boolean" usesScalarValueType="YES"/>
+        <attribute name="isLightning" optional="YES" attributeType="Boolean" usesScalarValueType="YES"/>
```

Comment:
The isLighnting boolean should not be saved to Core Data, it should be deleted from the node dictionary. FN no longer supports lightning.